packages/std/src/math/uint64.rs

```diff
@@ -210,6 +210,26 @@ impl<'a> ops::Add<&'a Uint64> for Uint64 {
     }
 }
 
+impl ops::Sub<Uint64> for Uint64 {
+    type Output = Self;
+
+    fn sub(self, rhs: Self) -> Self {
+        Uint64(
+            self.u64()
+                .checked_sub(rhs.u64())
+                .expect("attempt to subtract with overflow"),
```

Comment:
No no, I mean strictly cases like here - but I just realized, that `.expect` takes `&str` as an argument.
So we can't do, for example, `format!()` here to create some nice String with local variables (at least not cheap - without any copy etc).